sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -340,6 +340,11 @@ static void _send_to_iface(kernel_pid_t iface, gnrc_pktsnip_t *pkt)
         gnrc_pktbuf_release(pkt);
         return;
     }
+#ifdef MODULE_NETSTATS_IPV6
+    if_entry->stats.tx_success++;
+    if_entry->stats.tx_bytes += gnrc_pkt_len(pkt->next);
```

Comment:
IPv6 header length + ipv6->len; that would be. `pkt->next` may contain the netif header that isn't really data that is send over the interface.